JASP-Engine/JASP/R/ttestonesample.R

```diff
@@ -304,24 +314,24 @@ TTestOneSample <- function(jaspResults, dataset = NULL, options, ...) {
   }
   
   ## same for all tests
-  p     <- as.numeric(r$p.value)
-  stat  <- as.numeric(r$statistic)
-  m     <- as.numeric(r$estimate - r$null.value)
-  ciLow <- as.numeric(r$conf.int[1] - r$null.value)
-  ciUp  <- as.numeric(r$conf.int[2] - r$null.value)
+  p     <- as.numeric(tempResult[["p.value"]])
+  stat  <- as.numeric(tempResult[["statistic"]])
+  m     <- as.numeric(tempResult[["estimate"]]) 
+  ciLow <- as.numeric(tempResult[["conf.int"]][1])
+  ciUp  <- as.numeric(tempResult[["conf.int"]][2])
   ciLowEffSize <- as.numeric(confIntEffSize[1])
   ciUpEffSize  <- as.numeric(confIntEffSize[2])
   
   if (suppressWarnings(is.na(t)))  # do not throw warning when test stat is not 't' 
     stop("data are essentially constant")
   
-  result <- list(df = df, p = p, m = m, d = d, 
-              lowerCIlocationParameter = ciLow, upperCIlocationParameter = ciUp, 
-              lowerCIeffectSize = ciLowEffSize, upperCIeffectSize = ciUpEffSize)
+  result <- list("df" = df, "p" = p, "m" = m, "d" = d, 
+                 "lowerCIlocationParameter" = ciLow, "upperCIlocationParameter" = ciUp, 
+                 "lowerCIeffectSize" = ciLowEffSize, "upperCIeffectSize" = ciUpEffSize)
   result[[testStat]] <- stat
   
-  if (options$VovkSellkeMPR)
-    result[["VovkSellkeMPR"]] <- .VovkSellkeMPR(p)
+  if (options[["VovkSellkeMPR"]])
+    result[["VovkSellkeMPR"]] <- JASP:::.VovkSellkeMPR(p)
```

Comment:
The namespace ::: was needed to get things to work in the developer module. I suspect that we can remove `JASP:::` but I can't check it as I'm unable to build.